src/middleware/users.middleware.js

```diff
@@ -0,0 +1,34 @@
+import BaseRepository from '../repository/base.repository';
+import db from '../database/models';
+import responseGenerator from '../helpers/responseGenerator';
+
+export const userExist = async (req, res, next) => {
+  const { followeeId } = req.body;
+
+  const theUser = await BaseRepository.findOneByField(db.User, {
+    id: followeeId
+  });
+  if (!theUser) {
+    return responseGenerator.sendError(
+      res,
+      400,
+      `There is no user with id = ${followeeId}`
+    );
+  }
+  next();
+};
+
+export const isSelf = async (req, res, next) => {
```

Comment:
I like this. ❤️

Reply:
The `isSelf` variable declaration does clearly say what the variable is representing.

I think it will be better the variable declaration is self-descriptive since it is a middleware function that someone else on the team may need to make use of.

Reply:
> The `isSelf` variable declaration does clearly say what the variable is representing.
> 
> It will be better the variable declaration is self-descriptive since it is a middleware function that someone else on the team may need to make use of.

Thank you for your response @tobechukwuobitube , could you suggest a better name that I could use?